src/schema/objects/suffixes.yaml

```
@@ -670,6 +670,23 @@ nirs:
   value: nirs
   display_name: Near Infrared Spectroscopy
   description: Data associated with a Shared Near Infrared Spectroscopy Format file.
+noRF:
+  value: noRF
+  display_name: No Radio Frequency Excitation Scan
+  description: |
+    An MR noise scan often acquired alongside another MR scan.
+
+    These noise scans are acquired by disabling the radio frequency excitation,
+    while maintaining all other parameters from the associated scan.
+    These can be aquired using some other sequence with RF excitation removed
```

that is nice -- github actions/codespell annotates for a line, coolio, hence

```suggestion
    These can be acquired using some other sequence with RF excitation removed
```